models/wrf/shell_scripts/assimilate.csh

```diff
@@ -29,14 +29,13 @@ if ( $SUPER_PLATFORM == 'yellowstone' ) then
    setenv FORT_BUFFERED true
    mpirun.lsf ./filter || exit 1
 
-else if ( $SUPER_PLATFORM == 'cheyenne' ) then
+else if ( $SUPER_PLATFORM == 'derecho' ) then
 
-# TJH MPI_SHEPHERD TRUE may be a very bad thing
    setenv MPI_SHEPHERD FALSE
 
    setenv TMPDIR  /dev/shm
    limit stacksize unlimited
-   mpiexec_mpt dplace -s 1 ./filter || exit 1
+   mpiexec -n 256 -ppn 128 ./filter || exit 1
```

Comment:
Should this be 'mpiexec -n 128 -ppn 128' to match the other scripts?

Reply:
Actually never mind with this. This is actually set up in param.sh to use 2 nodes with filter. Correct me if I'm wrong, though.

Reply:
Yes -- that was my intention, which was to allocate for 2 nodes when executing filter, and only allocating for 1 node when calling for the wrf executables.  I was trying to be consistent with the previous scripting, but with customization for Derecho.  The explicit specification of processors and processors per nodes works, but I am not clear if Derecho requires the -n and -ppn flags to be declared explicitly for it to work.

Reply:
It is not required to use the -n and -ppn flags when you use mpiexec in a job submission script where you have it set with "#PBS -l select=2:ncpus=128:mpiprocs=128"

But it definitely doesn't hurt anything and I don't see any reason to remove it